Fixing builds on macos arm (#9)
elixir-nx/ex_faiss: README.md

ExFaiss will download, build, and cache Faiss on the first compilation. You must have CMake installed in order to build Faiss.

### MacOS Compilation

> For some reason ExFaiss doesn't want to build using the Apple LLVM so to make this work you will first need to install LLVM via [Homebrew](https://brew.sh/):

Reviewer: Also, fwiw, I am using Apple LLVM (macOS 12.x).

Author: For some reason on (macOS 13.3 (22E252)) it does not.

ExFaiss will download, build, and cache Faiss on the first compilation. You must have CMake installed in order to build Faiss.

> ### MacOS Compilation

Reviewer: ```suggestion
### macOS Compilation
```

> For some reason Faiss requires OpenMP to build and the built in Apple LLVM Clang compilers do not include it. So if you have troubles building using a mac you will need to install LLVM from homebrew.

Reviewer: ```suggestion

If you have troubles building on a macOS, you can try installing LLVM from homebrew.
```
